### PR TITLE
Docker tutorial

### PR DESCRIPTION
**Pull request by nebuk89**
Adding in V1 Docker tutorial to help with foundation of Docker use and highlighting unique experience between Docker and MSFT VS products.

### CODE REVIEW

**Comment by ktoliver** [comment]
Hi @nebuk89 - Thanks for your pull request. A member of the content team will review it.

File: docs/docker/tutorials/use-docker-compose.md [hunk 1]
```diff
@@ -237,7 +237,7 @@ Now that you have the `docker-compose.yml` file, you can start it up!
 
 1. First, make sure no other copies of the app and database are running (`docker ps` and `docker rm -f <ids>`).
 
-1. Start up the application stack using the `docker-compose up` command. Add the `-d` flag to run everything in the background.
+1. Start up the application stack using the `docker-compose up` command. Add the `-d` flag to run everything in the background. Alternatively you can right click on your Compose file and select the compose up option for the VSCode side bar.
```
Comment (ghogen) [inline docs/docker/tutorials/use-docker-compose.md:240]:
```suggestion
1. Start up the application stack using the `docker-compose up` command. Add the `-d` flag to run everything in the background. Alternatively, you can right-click on your Compose file and select the **Compose Up** option for the VS Code side bar. 
```

File: docs/docker/tutorials/persist-your-data.md [hunk 2]
```diff
@@ -129,7 +129,7 @@ The `Mountpoint` is the actual location on the disk where the data is stored. No
 
 > [!NOTE]
 > **Accessing Volume data directly on Docker Desktop**
-> While running in Docker Desktop, the Docker commands are actually running inside a small VM on your machine. If you wanted to look at the actual contents of the *Mountpoint* directory, you would need to first get inside of the VM.
+> While running in Docker Desktop, the Docker commands are actually running inside a small VM on your machine. If you wanted to look at the actual contents of the *Mountpoint* directory, you would need to first get inside of the VM. In WSL 2 this is inside a WSL 2 distro and can be accessed through the file explorer
```
Comment (ghogen) [inline docs/docker/tutorials/persist-your-data.md:132]:
```suggestion
> While running in Docker Desktop, the Docker commands are actually running inside a small VM on your machine. If you wanted to look at the actual contents of the *Mountpoint* directory, you would need to first get inside of the VM. In WSL 2, this is inside a WSL 2 distro and can be accessed through the File Explorer.
```

File: docs/docker/tutorials/deploy-to-cloud.md [hunk 3]
```diff
@@ -59,6 +59,18 @@ You can now see your ACI context listed and can right-click it to make it your c
 
 And, you can see that the container is running in a public IP and working correctly!
 
+1. Now we can have a look at our running container to see how it is working, we can start by having a look at the container logs
```
Comment (ghogen) [inline docs/docker/tutorials/deploy-to-cloud.md:62]:
```suggestion
1. Now, you can have a look at our running container to see how it is working. You can start by having a look at the container logs:
```

File: docs/docker/tutorials/deploy-to-cloud.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ You should only see your default context for local work.
 
 You'll need to set up an Azure account if you don’t already have one.
 
-1. Now you can create your new ACI context.
+1. Now you can create your new ACI context, you can do this by clicking the plus button on the contexts section of the Docker pannel
```
Comment (ghogen) [inline docs/docker/tutorials/deploy-to-cloud.md:31]:
```suggestion
1. Now you can create your new ACI context. You can do this by clicking the plus button on the contexts section of the Docker pannel
```

File: docs/docker/tutorials/deploy-to-cloud.md [hunk 5]
```diff
@@ -59,6 +59,18 @@ You can now see your ACI context listed and can right-click it to make it your c
 
 And, you can see that the container is running in a public IP and working correctly!
 
+1. Now we can have a look at our running container to see how it is working, we can start by having a look at the container logs
+ 
+ ```bash
+   docker logs distracted-jackson
+   ```
+
+1. We can also exec into our container as we would with a local container
```
Comment (ghogen) [inline docs/docker/tutorials/deploy-to-cloud.md:68]:
```suggestion
1. You can also exec into your container as you would with a local container.
```

File: docs/docker/tutorials/use-docker-compose.md [hunk 6]
```diff
@@ -275,22 +275,21 @@ Now that you have the `docker-compose.yml` file, you can start it up!
...
 
 If you twirl down the network, you will see the two containers you defined in the compose file.
 
-![VS Extension with app network expanded](media/vs-app-project-expanded.png)
+![VS Extension with Compose expanded](media/vs-app-project-expanded.png)
 
 ## Tear it all down
 
-When you're ready to tear it all down, simply run `docker-compose down`, or hit the trash can on the Docker Dashboard for the entire app. The containers will stop and the network will be removed.
+When you're ready to tear it all down, simply run `docker-compose down`, or right click on the application in the containers list in the VSCode Docker extension and select Compose down. The containers will stop and the network will be removed.
```
Comment (ghogen) [inline docs/docker/tutorials/use-docker-compose.md:288]:
```suggestion
When you're ready to tear it all down, simply run `docker-compose down`, or right-click on the application in the containers list in the VS Code Docker extension and select **Compose Down**. The containers will stop and the network will be removed.
```

**Comment by ktoliver** [comment]
@ghogen Would you like me to accept your revisions or leave it for @nebuk89 to update? Thanks.

**Comment by ghogen** [comment]
@ktoliver Yes, plz go ahead and accept those changes and then merge. Thanks!

**Pull request merged**